### PR TITLE
Fix examples for no-default-features

### PR DESCRIPTION
**Pull request by Luni-4**
Fixed tests for the `--no-default-features` option

Thanks in advance for your review! :)

### CODE REVIEW

File: examples/perf.rs [hunk 1]
```diff
@@ -7,20 +7,15 @@
...
-	}
-}
-
-use std::env;
 use lewton::VorbisError;
-use lewton::inside_ogg::OggStreamReader;
-use std::fs::File;
-use std::time::Instant;
 
+#[cfg(feature = "ogg")]
```
Comment (est31) [inline examples/perf.rs]: Can you instead put `#![cfg(feature="ogg")]` to the top of the file?

Comment (Luni-4) [reply examples/perf.rs]: I've tried, but `cargo` can't find the main function in that way

Comment (est31) [reply examples/perf.rs]: Oh I see. I think required-features is a better solution: https://doc.rust-lang.org/cargo/reference/manifest.html#the-required-features-field-optional

Comment (Luni-4) [reply examples/perf.rs]: Done, thanks!

**Pull request merged**